src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -55,13 +55,14 @@ object AstTools {
       astNode match {
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlStringType => WdlString(t.getSourceString)
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString)
+        case t: Terminal if t.getTerminalStr == "identifier" && wdlType == WdlStringType => WdlString(t.getSourceString)
```

Comment:
I'm not sure this is correct, I don't think identifiers should be converted to strings like this.  What problem did adding this line solve?